include/net/socket.h

```diff
@@ -175,6 +179,41 @@ static inline void freeaddrinfo(struct zsock_addrinfo *ai)
 #define EAI_AGAIN DNS_EAI_AGAIN
 #define EAI_FAIL DNS_EAI_FAIL
 #define EAI_NODATA DNS_EAI_NODATA
+
+/**
+ * @brief Access socket option values
+ *
+ * @param sock The socket that the operation refers to
+ * @param level Protocol level the option applies to
+ * @param optname Numerical value of the option to access
+ * @param optval Location, i.e. buffer, where to store the option value
+ * @param optlen Size of the option value location, i.e. buffer
+ *
+ * @return 0 on success, -1 on error with errno set to the error value
+ */
+static inline int getsockopt(int sock, int level, int optname,
+			     void *optval, socklen_t *optlen)
+{
+	return zsock_getsockopt(sock, level, optname, optval, optlen);
+}
+
+/**
+ * @brief Set socket option values
+ *
+ * @param sock The socket that the option refers to
+ * @param level Protocol level the option applies to
+ * @param optname Numerical value of the option to set
+ * @param optval Location, i.e. buffer, that contains the option value
+ * @param optlen Size of the option value location, i.e. buffer
+ *
+ * @return 0 on success, -1 on error with ernno set to the error value
```

Comment:
Typo: "ernno"

Reply:
Indeed. Fixed. Once I get the code around mbedTLS somehow on track, I'll also revisit configuration options in these four.